frontend/src/scenes/project-homepage/NewlySeenPersons.tsx

```diff
@@ -11,10 +11,18 @@ import { PersonType } from '~/types'
 import { ProfilePicture } from 'lib/components/ProfilePicture'
 import { asDisplay } from 'scenes/persons/PersonHeader'
 import { projectHomepageLogic } from './projectHomepageLogic'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
 
 function PersonRow({ person }: { person: PersonType }): JSX.Element {
     return (
-        <LemonButton fullWidth className="list-row" to={urls.person(person.distinct_ids[0])}>
+        <LemonButton
+            fullWidth
+            className="list-row"
+            to={urls.person(person.distinct_ids[0])}
+            onClick={() => {
+                eventUsageLogic.actions.reportPersonOpenedFromNewlySeenPersonsList()
```

Comment:
I think this is fine but just a heads up for the most part we follow a pattern of including these event usage logic action calls inside of the component's logic file instead. If we want to call this action from a component we should also probably stay consistent here with a useActions instead, like what we do normally 

`const { reportPersonOpenedFromNewlySeenPersonsList } = useActions(eventUsageLogic)`

Reply:
Got it, do you know if there's any advantage to going the `useActions` route? From looking through our repo, it looks like we're using both kind of interchangeably.

Reply:
Hmmmm maybe `useActions` guarantees that the logic is mounted?

Reply:
Yes I think so

Reply:
prob some patterns we should document

Reply:
Just flipped to the `useActions` pattern